tensorflow/cc/gradients/math_grad_test.cc

```diff
@@ -865,5 +865,14 @@ TEST_F(NaryGradTest, Minimum) {
   RunTest(x, x_init_value, y, shape);
 }
 
+TEST_F(NaryGradTest, Select) {
+  TensorShape x_shape({3, 4});
```

Comment:
Nit: Since x_shape is the same as y_shape, maybe rename one to 'shape' and use that everywhere?